soh/soh/SohMenuBar.cpp

```diff
@@ -1192,6 +1192,15 @@ void DrawEnhancementsMenu() {
                 "- Blue Warps\n"
                 "- Darunia\n"
                 "- Gold Skulltulas");
+            if (UIWidgets::PaddedEnhancementCheckbox("Show Age-Dependent Equipment", CVAR_ENHANCEMENT("EquimentAlwaysVisible"), true,
```

Comment:
Typo: `Equiment` &#8594; `Equipment`

This one appears multiple times in the added `CVAR_ENHANCEMENT` brackets.